src/main/java/org/phyloref/jphyloref/commands/TestCommand.java

```diff
@@ -112,6 +113,12 @@ public int execute(CommandLine cmdLine) throws RuntimeException {
       throw new IllegalArgumentException("Error: no input ontology specified (use '-i input.owl')");
     }
 
+    // If the input filename is '-', we should read the ontology from STDIN instead.
+    InputStream inputStreamToReadFrom = null;
+    if (str_input.equals("-")) {
+      inputStreamToReadFrom = System.in;
+    }
+
```

Comment:
Why not handle this such that from this point forward you can assume that `inputStreamToReadFrom` is set correctly -- either to `System.in` or to a `FileReader()`. Then you don't have to duplicate the nearly exact same logic for reading further down.

Reply:
D'oh! Fixed in 5b65728.